app/services/research_safety_adapters/scishield_training_synchronizer.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ResearchSafetyAdapters
+
+  class ScishieldTrainingSynchronizer
+    def initialize
+      @users = User.active.select(:id, :email)
```

Comment:
Why are you just grabbing id and email here?  Is there a performance concern?  

We might want to know the username at some point.  Also, `ScishieldApiAdapter` depends on this data including everything it needs to know about users.  Right now that's not encoded anywhere.  I'd vote for either just grabbing all the user data like `User.active` or defining the list of attributes in `ScishieldApiAdapter` and the using it here like:
`@users = User.active.select(ScishieldApiAdapter.user_attributes)`.

Reply:
Yeah, I was thinking about performance